setup.py

```diff
@@ -615,6 +615,18 @@ def add_data_files(config):
             if folder in dirs:
                 dirs.remove(folder)
 
+    # Force include the contents of some directories.
+    FORCE_INCLUDE_DIRS = [
+        os.path.join(SETUP_DIRECTORY, 'obspy', 'io', 'mseed', 'src',
+                     'libmseed', 'test')]
+
+    for folder in FORCE_INCLUDE_DIRS:
+        for root, _, files in os.walk(folder):
+            for filename in files:
+                config.add_data_files(
+                    os.path.relpath(os.path.join(root, filename),
+                                    SETUP_DIRECTORY))
```

Comment:
@krischer I just noticed that these files are missing in deb packaging.. can we somehow avoid using test data that are not in the usual location? To include this additional directory will be some additional work for all @obspy/packagers..